src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Architecture.cs

```diff
@@ -50,5 +50,20 @@ public enum Architecture
         /// A PowerPC 64-bit (little-endian) processor architecture.
         /// </summary>
         Ppc64le,
+        /// <summary>
+        /// RISCV32
+        /// </summary>
+        RiscV32,
+        /// <summary>
+        /// RISCV64
+        /// </summary>
+        /// <remarks>
+        /// This value indicates RV64GC set of extensions.
+        /// </remarks>
+        RiscV64,
```

Comment:
The approved API proposal only has RiscV64 in this enum.

https://github.com/dotnet/runtime/issues/73437#issuecomment-1716122572